### PR TITLE
Add PenguinMod

### PR DESCRIPTION
**Pull request by FreshPenguin112**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://penguinmod.github.io/PenguinMod-Gui/editor.html
Just a cool Scratch mod

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Review by FreshPenguin112 — COMMENTED** [review]
👍
That change is fine

**Comment by WilliamDavidHarrison** [comment]
> 👍 That change is fine

Why are you reviewing your own PR?

File: domains/penguinmod.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+ "owner": {
+   "username": "FreshPenguin112",
+   "email": "freshpenguin1@outlook.com"
+ },
+ "record": {
+   "URL": "penguinmod.github.io/penguinmod-gui/editor.html"
```
Comment (WilliamDavidHarrison) [inline domains/penguinmod.json:7]:
```suggestion
   "URL": "https://penguinmod.github.io/penguinmod-gui/editor.html"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Review by wdhdev — CHANGES_REQUESTED** [review]
I cant reach your website

**Comment by FreshPenguin112** [comment]
Fixed
Forgot some capitalization lol

**Review by wdhdev — CHANGES_REQUESTED** [review]
Still a 404

**Comment by FreshPenguin112** [comment]
fixed forgot capital M lmao

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — APPROVED** [review]
scratch clone? :eyes:

**Pull request merged**